src/Files.App/Actions/Content/Archives/Decompress/DecompressArchiveToChildFolderAction.cs

```diff
@@ -40,7 +40,7 @@ protected override void Context_PropertyChanged(object? sender, PropertyChangedE
 		private string ComputeLabel()
 		{
 			if (context.SelectedItems == null || context.SelectedItems.Count == 0)
-				return string.Empty;
+				return string.Format("BaseLayoutItemContextFlyoutExtractToChildFolder".GetLocalizedResource(), string.Empty);
```

Comment:
What does this change?

Reply:
It was empty string when no context given, like in this page.

Reply:
What does it look like now?

Reply:
![image](https://github.com/files-community/Files/assets/62196528/2641364a-608e-4763-9f0a-3b806b2b2abf)

Looks not nice but better than nothing to show. It'd be better if we add simple title for this situation too as you suggested for Install commands.